spec/support/schemas/user_loa1.json

```diff
@@ -1,6 +1,4 @@
 {
-  "$schema" : "http://json-schema.org/draft-04/schema#",
-  "type": "object",
```

Comment:
Was the removal of these lines intentional?

Reply:
nope, I'll fix